### PR TITLE
Add example i3status-rs config to README.md

### PR DESCRIPTION
**Pull request by bim9262**
I just added a swaync driver to i3status-rs and just wanted to share an example config.

### CODE REVIEW

**Comment by ErikReider** [comment]
I didn't even know that i3status worked with wlroots WMs

**Comment by ErikReider** [comment]
Oh, you're referring to i3status and not i3bar... My bad 😅

File: README.md [hunk 1]
```diff
@@ -240,6 +240,25 @@ ninja -C build
...
+block = "notify"
+format = " $icon {($notification_count.eng(1)) |}"
+driver = "swaync"
+[[block.click]]
+button = "left"
+action = "show"
+[[block.click]]
+button = "right"
+action = "toggle_paused"
+```
```
Comment (ErikReider) [inline README.md:260]: Place this below the Waybar example due to it being more popular

**Review by ErikReider — CHANGES_REQUESTED** [review]
Other than that, LGTM! :)

**Review by ErikReider — APPROVED** [review]
> I just added a swaync driver to i3status-rs

Oh wait, you actually added it into i3status?! LGTM. Thanks for the help ❤️

**Comment by bim9262** [comment]
I did (well i3status-rs)! You're welcome :smile:

**Pull request merged**